KCSForm-iOS/Classes/Cells/TagCell.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  TagCell.swift
+//  KCSForm-iOS
+//
+//  Created by Matthew Patience on 2020-02-13.
+//
+
+import UIKit
+
+class TagCell: FormCell {
+    
+    @IBOutlet var label: UILabel!
+    private var widthConstraint: NSLayoutConstraint!
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        self.label.font = FormStyle.shared.tagFont
+        updateStyle()
+        setSelected(false)
+        self.contentView.translatesAutoresizingMaskIntoConstraints = false
+        self.widthConstraint = self.contentView.widthAnchor.constraint(equalToConstant: 0)
+    }
+    
+    public func setCellWidth(_ width: CGFloat) {
+        self.widthConstraint.constant = width
+        self.widthConstraint.isActive = true
```

Comment:
Since we changed constant of a constraint, do we not need `layoutIfNeeded` here?

Reply:
No, cause this function gets called from the collectionview delegate when it's drawing the cell. So the cell doesn't exist yet. There won't ever be a time that this function gets called elsewhere. So I think it's fine.